src/sentry/models/release.py

```diff
@@ -330,7 +330,7 @@ def set_commits(self, commit_list):
         """
 
         # Sort commit list in reverse order
-        commit_list.sort(key=lambda commit: commit.get('date_added'), reverse=True)
+        commit_list.sort(key=lambda commit: commit.get('timestamp'), reverse=True)
```

Comment:
`date_added` looks like it's an attribute for the Commit model? https://github.com/getsentry/sentry/blob/master/src/sentry/models/commit.py#L17

Reply:
That's what I originally thought, but this list comes from https://github.com/getsentry/sentry/blob/master/src/sentry/api/serializers/rest_framework/commit.py

Reply:
Ahh, can we add a comment?